tests/test_builders/test_build_linkcheck.py

```diff
@@ -699,9 +708,9 @@ def test_connect_to_selfsigned_with_requests_env_var(monkeypatch, app):
     }
 
 
-@pytest.mark.sphinx('linkcheck', testroot='linkcheck-localserver-https', freshenv=True)
-def test_connect_to_selfsigned_nonexistent_cert_file(app):
-    app.config.tls_cacerts = "does/not/exist"
+@pytest.mark.sphinx('linkcheck', testroot='linkcheck-localserver-https', freshenv=True,
+                    confoverrides={'tls_cacerts': "does/not/exist"})
+def test_connect_to_selfsigned_nonexistent_cert_file(app: Sphinx) -> None:
```

Comment:
you *could* apply ruff/black-style formatting here as you go

Reply:
I'll keep that on the todo-list; I have #12126 open that changes a bunch of the `test_build_linkcheck` code, so I'd prefer not to introduce many-line conflicts (a few limited/construtive edit conflicts are OK, but reformatting looks like it'd be larger).